Addressing Issue #999 - [Accessibility] PeoplePicker doesn't have focus border when focused by keyboard

oengusmacinog opened the pull request:
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #999 
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Added onFocus state Boolean to BasePicker so a class can control the focus css on the input container div.

![peoplepicker-focus](https://user-images.githubusercontent.com/3977969/35306375-1f326418-0052-11e8-8b79-5fd34e635272.gif)

#### Focus areas to test

Checked on Tagpicker, DocumentPicker, and PeoplePicker.  I am unsure if there are any other instances but they should all inherit from BasePicker

joschect commented:
You'll just probably need to update the snaps and include a change file.

You can add a change file by running `rush change` in the cmd line at the root of this repo and following the prompts.

To update the snaps you'll change directories to `packages.office-ui-fabric-react` then do `npn run update-snapshots`

oengusmacinog commented:
added and emulated 'on focus' blue border to picker fields. This involved adding an onFocus variable to BasePicker state and added a control class to input container div
